src/controller/AutoCommissioner.cpp

```diff
@@ -156,10 +181,29 @@ Optional<System::Clock::Timeout> AutoCommissioner::GetCommandTimeout(Commissioni
 
 void AutoCommissioner::CommissioningStepFinished(CHIP_ERROR err, CommissioningDelegate::CommissioningReport report)
 {
-    if (report.stageCompleted == CommissioningStage::kFindOperational)
+    switch (report.stageCompleted)
     {
+    case CommissioningStage::kGetNetworkTechnology:
+        mNetworkTechnology.SetRaw(report.FeatureMap.features);
+        // Only one of these features can be set at a time.
```

Comment:
But multiple network technologies can be supported, just on different endpoints.  Worth a TODO comment about trying to handle that, at least.

Reply:
Right now they can't though - see discussion on the network cluster update here. https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/4738#pullrequestreview-827050074. As far as I know the device spec has not been updated to accommodate a new device type that holds a network commissioning cluster.

Reply:
People can make up manufacturer-specific device types that have a network commissioning cluster...

Reply:
I guess then the question is whether we want to support a manufacturing specific device type with an associated network cluster in the main SDK.

Reply:
We might at some point....  But to be clear, all I was asking for here was a TODO comment.

Reply:
Might not be too bad - I have a PR in the wings to read the descriptor/basic clusters, so once that goes in, this will be possible.

Reply:
Once this PR and https://github.com/project-chip/connectedhomeip/pull/14410 land, I can add extend the feature map checks to the other network clusters and select as appropriate.

Reply:
https://github.com/project-chip/connectedhomeip/issues/14412 to track - going to close this convo for now and address in a follow up.